packages/runtime/src/runtimes/react/components/draft-switcher/draft-switcher.tsx

```diff
@@ -35,8 +35,6 @@ export function DraftSwitcher({ isDraft }: { isDraft: boolean }) {
     }
   }, [showToolbar, shadowRoot, setShadowRoot])
 
-  if (!showToolbar) return null
-
```

Comment:
Always render the debug `<meta>` tag; toolbar visibility is already conditioned on `showToolbar` (below).